Formula/openlibm.rb

```diff
@@ -0,0 +1,31 @@
+class Openlibm < Formula
+  desc "High quality, portable, open source libm implementation"
+  homepage "https://openlibm.org"
+  url "https://github.com/JuliaMath/openlibm/archive/v0.7.0.tar.gz"
+  sha256 "1699f773198018b55b12631db9c1801fe3ed191e618a1ee1be743f4570ae06a3"
+
```

Comment:
```suggestion

keg_only :provided_by_macos

```

Reply:
Unless I'm greatly misunderstanding things, this definitely needs to be `keg_only` since libm is provided by macOS:

```
$ ls -l /usr/lib/libm.dylib
lrwxr-xr-x  1 root  wheel  15 Oct 23  2018 /usr/lib/libm.dylib -> libSystem.dylib
```

Reply:
I see, added.